pandas/tests/frame/test_indexing.py

```diff
@@ -3682,3 +3682,14 @@ def test_functions_no_warnings(self):
         with tm.assert_produces_warning(False):
             df['group'] = pd.cut(df.value, range(0, 105, 10), right=False,
                                  labels=labels)
+
+    def test_getitem_zerodim_np_array(self):
```

Comment:
I would rather have these added test in pandas/tests/indexing, in a sub-file

Reply:
Done!